docs/sources/setup-grafana/configure-security/configure-authentication/grafana-com/index.md

```diff
@@ -36,17 +36,16 @@ enabled = true
 ```
 
 ### Configure automatic login
+To automatically log in to Grafana using GrafanaCom authentication, set the `auto_login` option to `true`. This skips the login screen.
 
-Set `auto_login` option to true to attempt login automatically, skipping the login screen.
 This setting is ignored if multiple auth providers are configured to use auto login.
 
 ```
 auto_login = true
 ```
 
 ## Skip organization role sync
-
-To prevent the sync of org roles from Grafana.com, set `skip_org_role_sync` to `true`. This is useful if you want to manage the organization roles for your users from within Grafana.
+To prevent the synchronization of organization roles from Grafana.com, set the `skip_org_role_sync` option to `true`. This is useful if you want to manage the organization roles for your users from within Grafana.
```

Comment:
```suggestion
To prevent synchronizing organization roles from Grafana.com, set the `skip_org_role_sync` option to `true`. This is useful if you want to use Grafana to manage organization roles
```